lambdas/revoke-vcs/README.md

```diff
@@ -0,0 +1,14 @@
+# RevokeVcsHandler
+
+What this lambda is for and does: https://govukverify.atlassian.net/browse/PYIC-4555
+
+To run this lambda, create an appropriate payload including userIds with kbv vcs in the the appropriate environment.
```

Comment:
Here (and elsewhere) there are a few mentions of KBV specifically that we can probably make generic